constantinCurtis/src/main/java/data/collection/task/classes/Item.java

```diff
@@ -18,6 +20,39 @@ public String toString() {
                 "id=" + id +
                 ", title='" + title + '\'' +
                 ", type=" + type +
+                ", price=" + price +
                 '}';
     }
+
+    public long getId() {
+        return id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public Type getType() {
+        return type;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public static long getIdCounter() {
+        return idCounter;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public void setPrice(double price) {
```

Comment:
can we have a negative price? I think no? Could you handle this?

Reply:
done, thanks for advice.